app/src/main/java/com/example/levi9application/viewModel/CocktailViewModel.kt

```diff
@@ -31,18 +38,26 @@ class CocktailViewModel
 
     init {
         getCocktails()
+        val cocktailDAO = CocktailDatabase.getDatabase(application).getDao()
```

Comment:
We should not instantiate repo in the VM. Check comment above for the explanation and pattern.